test-assets/7677a9/slider.js

```diff
@@ -0,0 +1,44 @@
+function increaseSlider() {
+    document.getElementById('motionSlider').value++;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function decreaseSlider() {
+    document.getElementById('motionSlider').value--;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
```

Comment:
I'd factor that in a `changeSlider(value)` function. `increaseSlider()` and `decreaseSlider()` can still be here but would just call `changeSlider` with ±1.
Not really necessarily. I tend to factor in and abstract a lot 😁